effekt/shared/src/main/scala/effekt/context/Annotations.scala

```diff
@@ -467,7 +467,7 @@ trait TreeAnnotations { self: Context =>
       // addDefinedSymbolToSource(sym)
   }
 
-  def symbolOf(id: source.Id): Symbol = symbolOption(id) getOrElse {
+  def symbolOf(id: source.Id): Symbol = this.symbolOption(id) getOrElse {
```

Comment:
Is the `this.` here necessary? Shouldn't it refer to the same anyway?

Reply:
Yes, this seems to be necessary so the override works (otherwise there is an error from Scala about this being ambiguous)